app/controllers/items_controller.rb

```diff
@@ -9,6 +9,7 @@ class ItemsController < ApplicationController
   before_action :forbid_modify, :only => [
     :add_collection, :set_collection, :remove_collection,
     :datastream_update,
+    :embargo_update,
```

Comment:
Is this a functional change? Before, it looked like we restricted updating embargos to only `current_user.is_admin?`